k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
> AFAIK that essentially requires you to perform the build inside the Dockerfile

Reference: <https://docs.docker.com/docker-hub/builds/>

> It probably wouldn't be too time consuming to set up Travis CI with permissions to push to the Apereo group on Docker hub

Sounds good.
I've also heard good things about [using Circle CI to generate docker images](https://circleci.com/docs/2.0/building-docker-images/).
That is also what [CAS is currently using for CI](https://circleci.com/gh/apereo/cas), maybe @mmoayyed has some insight?